pkg/operator/controllers/imageconfig/image_controller.go

```diff
@@ -119,18 +119,20 @@ func (r *Reconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 // Switch case to ensure the correct registries are added depending on the cloud environment (Gov or Public cloud)
 func getCloudAwareRegistries(instance *arov1alpha1.Cluster) ([]string, error) {
-	var requiredRegistries []string
+	acrSubdomain := strings.Split(instance.Spec.ACRDomain, ".")[0]
+	var regionalRegistry string
+
 	switch instance.Spec.AZEnvironment {
 	case azureclient.PublicCloud.Environment.Name:
-		requiredRegistries = []string{instance.Spec.ACRDomain, "arosvc." + instance.Spec.Location + ".data." + azure.PublicCloud.ContainerRegistryDNSSuffix}
+		regionalRegistry = fmt.Sprintf("%s.%s.data.%s", acrSubdomain, instance.Spec.Location, azure.PublicCloud.ContainerRegistryDNSSuffix)
```

Comment:
technically we don't have ACR replication in all regions and environments, but to make this easy (and it has no additional impact including this) I'm +1 with this.

Reply:
Do we have any way of querying which environments have replication and which don't? If we do that could be included in the logic of the this function.

Reply:
no.  it's defined within the RP config which isn't present when we deploy the controllers into a cluster.  I think it's fine.